CHANGELOG.md

```diff
@@ -139,24 +144,26 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
 
 ### Emissary-ingress and Ambassador Edge Stack
 
-- Feature: It is now possible to use both the `getambassador.io/v2` and `getambassador.io/v3alpha1`
-  apiVersions.  This is accomplished with the addition of an `emissary-ingress-apiext` Service and
-  Deployment that handles converting between the two versions for the apiserver.  This allows the
-  user to semlessly request and author resources in whichever API version is desired, and
-  facilitates easier migration from Emissary 1.x to 2.x.  Resources authored in v3alpha1 will not be
-  visible to Emissary 1.x; resources authored in v2 will be visible to both Emissary 1.x and 2.x.
+- Change: Support for the Envoy V2 API is deprecated in Emissary-ingress v2.1, and will be removed
+  in Emissary-ingress v2.2.0. The `AMBASSADOR_ENVOY_API_VERSION` environment variable will be
+  removed at the same time. Only the Envoy V3 API will be supported (this has been the default since
+  Emissary-ingress v1.14.0).
+
+- Feature: Emissary-ingress supports `getambassador.io/v2` CRDs, to simplify migration from
+  Emissary-ingress 1.X. **Note:** it is important to read the <a
+  href="../topics/install/migration-matrix">migration documentation</a> before starting migration.
```

Comment:
This takes you to https://github.com/emissary-ingress/emissary/blob/topics/install/migration-matrix which does not exist.

Reply:
lmao

Reply:
```patch
diff --git a/docs/CHANGELOG.tpl b/docs/CHANGELOG.tpl
index 5f82b2329..999ea5a15 100644
--- a/docs/CHANGELOG.tpl
+++ b/docs/CHANGELOG.tpl
@@ -106,6 +106,7 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
     strings.ReplaceAll "</i>" "*" |
     strings.ReplaceAll "<code>" "`" |
     strings.ReplaceAll "</code>" "`" |
+    strings.ReplaceAll "href=\"../" "href=\"https://www.getambassador.io/docs/emissary/latest/" |
     strings.WordWrap 100 }}
 {{- end }}{{ end }}{{ end }}
 {{ if ne $release.date "N/A" }}
@@ -119,6 +120,7 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
     strings.ReplaceAll "</i>" "*" |
     strings.ReplaceAll "<code>" "`" |
     strings.ReplaceAll "</code>" "`" |
+    strings.ReplaceAll "href=\"../" "href=\"https://www.getambassador.io/docs/emissary/latest/" |
     strings.WordWrap 98 |
     strings.Indent 2 |
     strings.TrimPrefix "  " }}{{ if index . "github" }}{{ range .github }} ([{{.title}}]){{ end }}{{ end }}
@@ -135,6 +137,7 @@ Please see the [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest
     strings.ReplaceAll "</i>" "*" |
     strings.ReplaceAll "<code>" "`" |
     strings.ReplaceAll "</code>" "`" |
+    strings.ReplaceAll "href=\"../" "href=\"https://www.getambassador.io/docs/edge-stack/latest/" |
     strings.WordWrap 98 |
     strings.Indent 2 |
     strings.TrimPrefix "  " }}{{ if index . "github" }}{{ range .github }} ([{{.title}}]){{ end }}{{ end }}
```
aught to do it